### ISSUE TITLE
2.3.0 build issue

### ISSUE THREAD

**Issue by devrimgunduz** [plv8/plv8#255]
Hi,

When I run "make static",  I get:

make[1]: Makefile.v8: No such file or directory
make[1]: *** No rule to make target 'Makefile.v8'.  Stop.

What I am missing?

Regards, Devrim

**Comment by JerrySievert** [comment]
that you should no longer use `make static`.

I thought I had cleaned up all of the possible vectors for that, but I've obviously missed one.  I'll put an `echo` in there to inform people to use `make` instead.

see https://github.com/plv8/plv8/blob/master/Changes#L6 for the short version, and the docs have a longer version, as well as the notes on the original issue if you're looking for more context.

long story short:

```
$ make
```

if you somehow manage to get a modern version of v8 compiled into a shared library (good luck), then you can run `make shared`.
